tests/common/fixtures/conn_graph_facts.py

```diff
@@ -76,7 +76,7 @@ def get_graph_facts(duthost, localhost, hostnames):
                     kargs["hosts"] = hostnames
                 conn_graph_facts = localhost.conn_graph_facts(
                     **kargs)["ansible_facts"]
-                return conn_graph_facts
+                return key_convert2str(conn_graph_facts)
```

Comment:
Fixing from pytest fixture side only resolves the issue we encountered in pytest.
The conn_graph_facts ansible module may be called in ansible or other places. It would be better to resolve the encoding issue directly from the conn_graph_facts ansible module located at `sonic-mgmt/ansible/library/conn_graph_facts.py`